doc/classes/AudioStreamGenerator.xml

```diff
@@ -24,6 +25,39 @@
 		    for i in range(frames_available):
 		        playback.push_frame(Vector2.ONE * sin(phase * TAU))
 		        phase = fmod(phase + increment, 1.0)
+		[/gdscript]
+		[csharp]
+                [Export] public AudioStreamPlayer Player { get; set; }
```

Comment:
The codeblock should be indented with tabs like above (and then indentation of the code itself within the codeblock is made with 4 spaces).